README.md

```diff
@@ -1 +1,68 @@
-# Hello-World
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Hello World
+
+> This is first solo project for learnig purpose 
+
+
+## Built With
+
+- HTML
+- VS Code
+
+## Getting Started
+
+-I have just created repository in Github and clone it in my local machine to further changes
+
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+  -github basic commands
+  -linters
+  -Html 
+  -css
+  -basic coding 
+
+### Setup
+
+### Install
+
+### Usage
+
+### Run tests
+
+### Deployment
```

Comment:
- [ ] Kindly delete those sections that you are not using yet, for a professional README file. 👍🏽

Reply:
I have removed sectioned that are not using yet